cleanup.swift

```diff
@@ -47,13 +50,25 @@ func deleteDerivedDataFor(project: String) {
 let arguments = CommandLine.arguments
 let shouldDeleteDerivedData = arguments.contains("--clean-derived-data")
 
-for path in pathsToClean {
+let projectPath: String
+if
+    let pathIndex = arguments.firstIndex(of: "--path"),
+    arguments.count > pathIndex + 1
```

Comment:
Shouldn't this be `>=` if they only provide `--path`?

Reply:
This is to force a path to be passed in after `--path`, if we used `>=` and no actual path was passed in we'd have an out of bounds in `arguments[pathIndex + 1]`